core/src/main/java/org/apache/gravitino/proto/RoleEntitySerDe.java

```diff
@@ -44,7 +44,7 @@ public Role serialize(RoleEntity roleEntity) {
             .setName(roleEntity.name())
             .setAuditInfo(new AuditInfoSerDe().serialize(roleEntity.auditInfo()));
 
-    for (SecurableObject securableObject : roleEntity.securableObjects()) {
+    for (org.apache.gravitino.authorization.SecurableObject securableObject : roleEntity.securableObjects()) {
```

Comment:
SecurableObject could be import as well?

Reply:
Yes, you must use org.apache.gravitino.proto.SecurableObject rather than org.apache.gravitino.authorization.SecurableObject to function normally.

Reply:
Perhaps I'm missing something but why not import org.apache.gravitino.authorization.SecurableObject?